diff --git a/root/etc/uci-defaults/luci-homeproxy b/root/etc/uci-defaults/luci-homeproxy
index 70c34fe..723a7db 100755
--- a/root/etc/uci-defaults/luci-homeproxy
+++ b/root/etc/uci-defaults/luci-homeproxy
@@ -19,6 +19,12 @@ if ! uci -q batch <<-EOF >"/dev/null"
 	set firewall.homeproxy_input.position="chain-pre"
 	set firewall.homeproxy_input.chain="input"
 
+	delete firewall.homeproxy_client
+	set firewall.homeproxy_client=include
+	set firewall.homeproxy_client.type=nftables
+	set firewall.homeproxy_client.path="/var/run/homeproxy/fw4_client.nft"
+	set firewall.homeproxy_client.position="table-post"
+
 	delete firewall.homeproxy_post
 	commit firewall
 EOF
